frontend/src/templates/AFrameWrapper.jsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import 'aframe';
+
+function fixName(name) {
```

Comment:
i see this in 2 places, could probably extract this into a helper function

Reply:
Resolved: Saw this in 3 places and moved to util function + chaining

Reply:
Thanks!